app/models/order.rb

```diff
@@ -3,7 +3,12 @@
 class Order < ApplicationRecord
   belongs_to :employee
   belongs_to :reward
-  enum status: { undelivered: 0, delivered: 1 }
+  scope :filter_by_status, ->(status) { where(status: status) }
```

Comment:
it's a security concern - when you get a value from a user you should interpolate it into the SQL, so the `where` would look like this:

`where("status = ?", status)`